rustfmt.toml

```
@@ -0,0 +1,10 @@
+max_width = 100
+reorder_imports = true
+reorder_modules = true
+
+# == Nightly only. ==
+# imports_indent = "Block"
+# imports_layout = "Mixed"
+# imports_granularity = "Crate"
+# group_imports = "Preserve"
+# reorder_impl_items = false
```

We can probably use the settings from Agave
```suggestion
imports_granularity = "One"
group_imports = "One"
```